modules/programs/firefox/mkFirefoxModule.nix

```
@@ -964,6 +964,7 @@ in
       release = mkOptionDefault (builtins.head (lib.splitString "-" cfg.package.version));
 
       policies = {
+        NoDefaultBookmarks = lib.mkIf (builtins.any (profile: profile.bookmarks.enable) cfg.profiles) false;
```

Sorry, it's an attribute set
```suggestion
        NoDefaultBookmarks = lib.mkIf (builtins.any (profile: profile.bookmarks.enable) (builtins.attrValues cfg.profiles)) false;
```